TeamCode/src/main/java/org/firstinspires/ftc/teamcode/jacksonSama/ShooterOnly.java

```diff
@@ -73,13 +75,13 @@ public void initialize() {
                     }
                 }));
 
-        driveCommand = new Com_Drive(mecDrive, m_driverOp::getRightX, m_driverOp::getLeftY, m_driverOp::getLeftX);
+        driveCommand = new Com_Drive(mecDrive, m_driverOp::getLeftY, m_driverOp::getLeftX, m_driverOp::getLeftX);
 
         shooterSystem = new ShooterSystem(shot, telemetry, () -> pwrSelect);
         shootCommand = new Com_Shoot(shooterSystem);
         stopCommand = new Com_NoShoot(shooterSystem);
         toggleShooter = new GamepadButton(m_driverOp, GamepadKeys.Button.A)
-                .whenPressed(new ConditionalCommand(shootCommand, stopCommand, shooterSystem::active))
+                .whileHeld(new ConditionalCommand(shootCommand, stopCommand, shooterSystem::active))
                 .whenReleased(new InstantCommand(shooterSystem::toggle));
```

Comment:
Idea: we use the `toggleWhenPressed` binding instead.
```suggestion
                .toggleWhenPressed(shootCommand);
```
Then we can remove the active and toggle methods from the Subsystem and also not longer need the stop command.